### ISSUE TITLE
sysctl missing

### ISSUE THREAD

**Issue by fxrsliberty** [atomic-penguin/cookbook-nfs#109]
I have RHEL 7.4 with 
Chef Development Kit Version: 2.2.1
chef-client version: 13.3.42
delivery version: master (73ebb72a6c42b3d2ff5370c476be800fee7e5427)
berks version: 6.3.1
kitchen version: 1.17.0
inspec version: 1.35.1
I am trying to role out an NFS server and hopefully some clients. However,
the server.rb depends on sysctl cookbook which I can no longer get. 
Any work around?

**Comment by bish0polis** [comment]
I just grabbed sysctl 1.0.5 last night or so.  It should still be there.

I thought of just removing the dep in metadata and seeing how bad the wreckage was; I'm betting it'll just work, but not in Prod because #prod.  For now, I grabbed it, and I can deal with chef-14 complaining about it being there while the other cookbooks all update around the change.
````
cd ~/chef-repo
git submodule add \
    https://github.com/sous-chefs/sysctl.git \
    cookbooks/sysctl \
&& git add .gitmodules $_ \
&& git commit -m 'adding module' .gitmodules $_
````

And done!  Sooo easy to do the cheap workaround and get back to the main task, but I'm not sure whether that's sloth or pragmatism!

Roll/Role.  Heh.  I see what you did there.  :-D

**Comment by fxrsliberty** [comment]
Yeah, I got it too, I grabbed the code directly from GitHub too. Still
sucks that you can't "knife" it down...

**Comment by MarkGibbons** [comment]
sysctl 1.0.5 is on the supermarket.chef.io site.  You should be able to download it from there.

**Comment by atomic-penguin** [comment]
It is on the Supermarket site but probably will not work on Chef 14, as sysctl has been deprecated.  I am trying to find out what the right thing to do here, metadata wise, to maintain backwards compatibility.  That may mean a major version bump to officially support Chef 14.

**Comment by bish0polis** [comment]
> probably will not work on Chef 14

We're running Chef 14.2.0 here.

I pulled down the repo as a submodule under /cookbooks, pushed it to my chefbox, and carried on (with nfs still depending on it).  It'll whine, but if we're eating the brownies before they're baked we should expect some softness in the center.

It'd be neat if things rolled into chef showed up as virtual-provides to satisfy the depends requirements.  We've got a great, long and successful history of virtual-provides and obsoletes working really well in RPM-land.
